modules/server/src/main/scala/higherkindness/compendium/migrations/Migrations.scala

```diff
@@ -17,18 +17,25 @@
 package higherkindness.compendium.migrations
 
 import cats.effect.Sync
-import cats.implicits._
+import cats.syntax.applicativeError._
```

Comment:
I'm not against it, but I do not see the need to use multiple `cats.syntax` imports instead of just using `cats.implicits._`, which is how we usually operate.

Reply:
Ok! I prefer to use specific imports, but it makes sense to use the same import strategy across projects, to be consistent. I'll change those.